docs/changelog/changelog.rst

```diff
@@ -32,6 +32,9 @@ New Features
   If the full 3D distance is desired, set the ``3dDistance`` flag in the 
   configuration block of the :ref:`distance-task`.
   (`#360 <https://github.com/BYU-PCCL/holodeck/issues/360>`_)
+- :meth:`~holodeck.weather.WeatherController.set_weather` now has sunny
```

Comment:
Proposed edit:
`Agents can see clearly now, the rain can be gone`

Reply:
https://www.youtube.com/watch?v=nwNl2FNzCpk